manager.go

```diff
@@ -157,19 +157,22 @@ func (m *Manager) Restart() (*os.Process, error) {
 	cmd.Stdin = os.Stdin
 	cmd.Env = env
 
-	pr, pw, err := os.Pipe()
+	pipeRead, pipeWrite, err := os.Pipe()
```

Comment:
Any reason for the var name change, doesn't seem needed?

Reply:
I didn't know what pr or pw were when I first read this, so I clarified them for readability. Was personal preference having more explicit variable names

Reply:
We tend to push for short names, as it helps prevent the issues detailed above with excessive line lengths.

If you don't know what the function is returning then that's a lack of knowledge which should just be improved instead of using longer names. e.g.
```
n, err := buf.Write(data)
```
If you don't know ```n``` is a count then your should look at the docs, and not convert the above to:
```
writtenCount, err := buf.Write(data)
```

Short var names are typically idiomatic, that doesn't mean there's a one size fits all rule, but in this case the short names are the right decision.

Reply:
I did figure out what to name the variables by just looking up what the function did, so I guess at that point should have realised that I didn't need to rename them from everyone! Will revert back to what they were